lib/fog/kubevirt/compute/models/persistentvolumes.rb

```diff
@@ -35,6 +35,11 @@ def create(args = {})
           name = args[:name]
           labels = args[:labels]
 
+          # type is required
+          if args[:type].nil? || args[:type].empty?
+            raise ::Fog::Kubevirt::Errors::ValidationError
```

Comment:
Please add an error message to the exception so it will be clear to the client why this request isn't valid

Reply:
Ack